### PR TITLE
LPS-191479 - Normalized URL doesn't point to the same source as the accented version did

### PR DESCRIPTION
**Pull request by henriquelimadotcom**
**Ticket:** https://liferay.atlassian.net/browse/LPS-191479
@dacousalr 

# Problem Description
Previously, it was possible to create web content containing accented links in the application. However, due to recent changes, this functionality was removed, resulting in broken accented links for customers who created content during the period when this option was available. This situation has caused disruptions and a negative impact on the user experience, as well as the integrity of friendly URLs.

# Solution Objective
The proposed solution aims to restore the functionality of the existing accented links, allowing them to function correctly, even though it will no longer be possible to create new links with accents in the future. The focus is on ensuring the continuity of the integrity of friendly URLs and providing a consistent user experience.

### CODE REVIEW

**Comment by liferay-lima** [comment]
ci:test:sf

**Comment by liferay-lima** [comment]
ci:test:relevant

**Comment by AliciaGarciaGarcia** [comment]
ci:test:sf

**Comment by AliciaGarciaGarcia** [comment]
ci:test:friendly-url

**Comment by AliciaGarciaGarcia** [comment]
ci:test:relevant

**Comment by Jeremy-Ch** [comment]
The failures are unrelated.

**Comment by Jeremy-Ch** [comment]
ci:test:relevant

**Comment by AliciaGarciaGarcia** [comment]
ci:forward

**Comment by Jeremy-Ch** [comment]
ci:forward

**Comment by Jeremy-Ch** [comment]
ci:test:relevant

File: modules/apps/friendly-url/friendly-url-api/src/main/java/com/liferay/friendly/url/service/FriendlyURLEntryLocalService.java [hunk 1]
```diff
@@ -251,10 +249,19 @@ public long dynamicQueryCount(
 	public FriendlyURLEntry fetchFriendlyURLEntry(
 		long groupId, Class<?> clazz, String urlTitle);
 
+	@Transactional(propagation = Propagation.SUPPORTS, readOnly = true)
+	public FriendlyURLEntry fetchFriendlyURLEntry(
+		long groupId, Class<?> clazz, String urlTitle, boolean normalizeWithAccent);
```
Comment (AliciaGarciaGarcia) [inline modules/apps/friendly-url/friendly-url-api/src/main/java/com/liferay/friendly/url/service/FriendlyURLEntryLocalService.java:254]:
```
     [echo] --- a/modules/apps/friendly-url/friendly-url-api/src/main/java/com/liferay/friendly/url/service/FriendlyURLEntryLocalService.java
     [echo] +++ b/modules/apps/friendly-url/friendly-url-api/src/main/java/com/liferay/friendly/url/service/FriendlyURLEntryLocalService.java
     [echo] @@ -31,12 +31,14 @@ import com.liferay.portal.kernel.transaction.Isolation;
     [echo]  import com.liferay.portal.kernel.transaction.Propagation;
     [echo]  import com.liferay.portal.kernel.transaction.Transactional;
     [echo]  import com.liferay.portal.kernel.util.OrderByComparator;
     [echo] -import org.osgi.annotation.versioning.ProviderType;
     [echo]  
```
Is relevant 

build Service needed here

**Review by AliciaGarciaGarcia — CHANGES_REQUESTED** [review]

**Comment by AliciaGarciaGarcia** [comment]
ci:test:sf

**Comment by henriquelimadotcom** [comment]
ci:test:sf

**Review by AliciaGarciaGarcia — APPROVED** [review]

**Comment by AliciaGarciaGarcia** [comment]
ci:forward

**Comment by WYZ-Issac** [comment]
ci:test:relevant

**Comment by WYZ-Issac** [comment]
ci:forward

**Pull request closed without merging**